obs-studio-server/source/nodeobs_api.cpp

```diff
@@ -328,6 +329,9 @@ static void                                    node_obs_log(int log_level, const
 			// File Log
 			*logStream << newmsg << std::flush;
 
+            // Internal Log
+			logMessages.push_back(newmsg);
```

Comment:
The reason it is 150 is that Sentry doesn't handle well more then 150 lines when processing annotations, it messes its JSON formation and hides most on the information cell. This log is supposed to be used for other purposes then reports so it's better to not limit it (at least right now).
I'm still in the process to find a way to increase the Sentry limit (if possible to support the entire log).

Reply:
I still think there should be some limit if we keep log in ram. Users run program for days some time and without limit this log will only grow till no free memory left. 
Also log still saved to filestream in `*logStream << newmsg << std::flush;` and can be read from there if needed more for some other purpose.

Reply:
I've updated how the log stuff is done, I'm also limiting the total amount of messages stored by 150 but the error and warnings are still stored in a vector (since their number won't be huge).

The problem using the file is that some users aren't being able to save the log file, on these cases it would be impossible to use the file itself, but for now I will limit the amount of data stores and in revise this in the future, when/if necessary.